src/main/java/com/iexec/core/chain/adapter/BlockchainAdapterService.java

```diff
@@ -165,4 +169,27 @@ public BlockchainAdapterService(BlockchainAdapterClient blockchainAdapterClient)
         return Optional.empty();
     }
 
+    /**
+     * Retrieve and store the public chain config.
+     */
+    public PublicChainConfig getPublicChainConfig() {
+        if (publicChainConfig != null) {
+            return publicChainConfig;
+        }
+
+        try {
+            final ResponseEntity<PublicChainConfig> response =
+                    blockchainAdapterClient.getPublicChainConfig();
+            if (response.getStatusCode().is2xxSuccessful() && response.hasBody()) {
+                final PublicChainConfig config = response.getBody();
```

Comment:
What's the purpose of this local variable?

Reply:
Well... Nothing :)
Thanks!